logic/src/commonMain/kotlin/com/wire/kalium/logic/data/conversation/ConversationRepository.kt

```diff
@@ -54,6 +55,10 @@ interface ConversationRepository {
     suspend fun deleteMember(conversationID: QualifiedIDEntity, userID: QualifiedIDEntity): Either<CoreFailure, Unit>
     suspend fun getOneToOneConversationDetailsByUserId(otherUserId: UserId): Either<CoreFailure, ConversationDetails.OneOne?>
     suspend fun createGroupConversation(name: String? = null, members: List<Member>, options: ConverationOptions = ConverationOptions()): Either<CoreFailure, Conversation>
+    suspend fun getConversationsForNotifications(): Flow<List<Conversation>>
+    suspend fun setConversationAsNonNotified(qualifiedID: QualifiedID): Either<StorageFailure, Unit>
+    suspend fun setConversationAsNotified(qualifiedID: QualifiedID, date: String): Either<StorageFailure, Unit>
```

Comment:
Actually if you introduce the `lastModifiedDate` you don't need pass the date at all, because then you can do: `last_notified_date = last_modified_date`

Reply:
what is the `lastModifiedDate` ? 
if it's a date when the changes to the DB were done, then it doesn't help me. Cause it will be updated every time we insert a new message into DB, so there will never be notifications.

Reply:
I explain it another comment.